### ISSUE TITLE
Send notification with the result of execution

### ISSUE THREAD

**Issue by tomaswarynyca** [metroline/metroline#24]
We could add an option that at the end of the execution of the work a notification is sent to x platform such as telegram, slack, discord indicating what was the result of the execution, if everything was executed correctly or if we had a problem that did not allow the complete execution.

**Comment by tomaswarynyca** [comment]
Yes, as you say, it should be global to cover as many services as possible.
I will analyze what you have put together at Meli and see what options I can come up with to implement.

**Comment by gempain** [comment]
Awesome ! By the way, thanks so much for putting effort into improving this project. I'm happy to see interest and PRs coming in, it's great to see other people collaborating !

**Comment by 12nick12** [comment]
Just here to add this would be awesome. That's really the only thing I see missing coming from Drone (I wasn't a heavy user, but email notifications of completion/failure were nice).
